src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,29 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    private List<Truck> trucks = new ArrayList<>();
+
+    @Override
+    public List<Truck> get() {
+        Truck truck1 = new Truck();
+        Truck truck2 = new Truck();
+        Truck truck3 = new Truck();
+        trucks.add(truck1);
+        trucks.add(truck2);
+        trucks.add(truck3);
+        truck1.setColor("RED");
+        truck1.setName("Car transporters");
+        truck1.setCurbWeight(5000);
+        truck2.setColor("BLUE");
+        truck2.setName("Cement truck");
+        truck2.setCurbWeight(8000);
+        truck3.setColor("YELLOW");
+        truck3.setName("Box truck");
+        truck3.setCurbWeight(2500);
+        return trucks;
```

Comment:
Consider creating a local variable in your implementation of `get()` method. Instantiating and populating the 'trucks' field directly is not a good practice as it keeps state between invocations of `get()`. Instead, create a new local list inside the method, populate it with trucks, and return that new list.